src/cosmoz-tab-card.js

```diff
@@ -60,19 +64,29 @@ const CosmozTabCard = ({ heading }) =>
 		</style>
 
 		<div id="header" part="header">
+			${collapsable
+				? html`<input
+						type="checkbox"
+						@change=${() => setCollapsed((c) => !c)}
+				  />`
+				: nothing}
 			<h1 class="heading" part="heading">
 				${heading}<slot name="after-title"></slot>
 			</h1>
 			<slot name="card-actions"></slot>
 		</div>
 
 		<div id="content" part="content">
-			<slot></slot>
+			${collapsable
+				? html`<cosmoz-collapse ?opened=${collapsed}
+						><slot></slot
+				  ></cosmoz-collapse>`
+				: html`<slot></slot>`}
```

Comment:
You can always put the cosmoz-collapse there, just don't use it.

```suggestion
			<cosmoz-collapse ?opened=${collapsed}
						><slot></slot
				  ></cosmoz-collapse>
```

Reply:
Updated. Also, I needed to invert the condition (equal `opened` to the contrary of `isCollapsed`), otherwise the non collapsable card would appear collapsed.